modules/support-submitting-a-case.adoc

```diff
@@ -8,7 +8,7 @@
 .Prerequisites
 * You have access to the cluster.
 * You have a Red Hat Customer Portal account.
-* You have a Red Hat standard or premium subscription.
+* You have a Red Hat link:https://www.redhat.com/en/technologies/cloud-computing/openshift/platform-plus[OpenShift Platform Plus] subscription.
```

Comment:
You update this line so that the link spans across the whole product name "Red Hat OpenShift Platform Plus". For example:

> * You have a link:https://www.redhat.com/en/technologies/cloud-computing/openshift/platform-plus[Red Hat OpenShift Platform Plus] subscription.

Alternatively, you could provide the link for further reading in a separate sentence. For example:

> * You have a Red Hat OpenShift Platform Plus subscription. For more information, see link:https://www.redhat.com/en/technologies/cloud-computing/openshift/platform-plus[Red Hat OpenShift Platform Plus].